Source/PTEDashboard.m

```diff
@@ -59,9 +59,9 @@ - (instancetype)initWithFrame:(CGRect)frame
         }
         
         // Load Storyboard
-        self.rootViewController = [[UIStoryboard storyboardWithName:@"LumberjackConsole"
-                                                             bundle:nil] instantiateInitialViewController];
-        
+			  UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"LumberjackConsole" bundle:[NSBundle bundleForClass:[self class]]];
+			  self.rootViewController = [storyboard instantiateInitialViewController];
+			
```

Comment:
Yes, probably mines are the default ones. Could you just fix them so I can merge it?